torch/csrc/profiler/combined_traceback.h

```diff
@@ -58,15 +58,19 @@ struct TORCH_API CapturedTraceback : public c10::GatheredContext {
   int traversePython(visitproc visit, void* arg);
   int clearPython();
 
-  void recordUserDefinedFrame(const unwind::Frame& frame) {
-    user_defined_frames_.push_back(frame);
+  void recordUserDefinedMetadata(std::string name, std::string value) {
```

Comment:
It doesn't really make sense to put metadata here. This is suppose to be a unified traceback object. Adding extra frames to it made sense, but arbitrary extra data will be hard to treat consistently.

Reply:
Makes sense, I've removed user_defined frames from traceback object now.